src/table/utils/__tests__/copy-utils.spec.ts

```diff
@@ -4,26 +4,128 @@ describe('copyCellValue: ', () => {
+    elementClass = 'sn-table-head-cell';
+    await copyCellValue(evt);
+
+    expect(global.console.log).toHaveBeenCalledTimes(0);
+    expect(writeMock).toHaveBeenCalledTimes(1);
+    expect(writeMock).toHaveBeenCalledWith('head text');
   });
 
-  it('should not send console error when copying cell values', () => {
+  it('should call clipboard.writeText but not console log for body cell', async () => {
     evt = {
       target: {
-        children: [],
+        firstChild: {
+          textContent: 'body text',
+        },
+        classList,
       } as unknown as HTMLElement,
     };
-    Object.assign(navigator, {
-      clipboard: {
-        writeText: jest.fn().mockReturnValueOnce(Promise.resolve()),
-      },
+    elementClass = 'sn-table-cell';
+    await copyCellValue(evt);
+
+    expect(global.console.log).toHaveBeenCalledTimes(0);
+    expect(writeMock).toHaveBeenCalledTimes(1);
+    expect(writeMock).toHaveBeenCalledWith('body text');
+  });
+
+  it('should call clipboard.writeText but not console log for span in head cell (sort label)', async () => {
+    evt = {
+      target: {
+        firstChild: {
+          textContent: 'span text',
+        },
+        tagName: 'SPAN',
+        classList,
+      } as unknown as HTMLElement,
+    };
+    await copyCellValue(evt);
+
+    expect(global.console.log).toHaveBeenCalledTimes(0);
+    expect(writeMock).toHaveBeenCalledTimes(1);
+    expect(writeMock).toHaveBeenCalledWith('span text');
+  });
+
+  it('should call clipboard.writeText but not console log for svg in head cell (sort icon)', async () => {
+    evt = {
+      target: {
+        parentElement: {
+          firstChild: {
+            textContent: 'svg text',
+          },
+        },
+        tagName: 'svg',
+        classList,
+      } as unknown as HTMLElement,
+    };
+    await copyCellValue(evt);
+
+    expect(global.console.log).toHaveBeenCalledTimes(0);
+    expect(writeMock).toHaveBeenCalledTimes(1);
+    expect(writeMock).toHaveBeenCalledWith('svg text');
+  });
+
+  it('should not call clipboard.writeText nor console log when value is not defined', async () => {
+    evt = {
+      target: {
+        classList,
+      } as unknown as HTMLElement,
+    };
+    await copyCellValue(evt);
+
+    expect(global.console.log).toHaveBeenCalledTimes(0);
+    expect(writeMock).toHaveBeenCalledTimes(0);
+  });
+
+  it('should call console.log when writeText throws error', async () => {
+    writeMock = jest.fn().mockImplementation(() => {
+      throw new Error();
     });
-    copyCellValue(evt);
-    expect(console.log).not.toHaveBeenCalled();
+    evt = {
+      target: {
+        parentElement: {
+          firstChild: {
+            textContent: 'svg text',
+          },
+        },
+        tagName: 'svg',
+        classList,
+      } as unknown as HTMLElement,
+    };
+    await copyCellValue(evt);
+
+    expect(global.console.log).toHaveBeenCalledTimes(0);
+    expect(writeMock).toHaveBeenCalledTimes(0);
```

Comment:
@haxxmaxx ... but the test explanation says console.log should be called

Reply:
ah yes sloppy. They both should be called. I need to add it to clipboard again. I just pushed the fix